static/js/components/SourceTableFilterForm.jsx

```diff
@@ -21,6 +21,7 @@ import Button from "./Button";
 import { allowedClasses } from "./ClassificationForm";
 
 import * as gcnEventsActions from "../ducks/gcnEvents";
+import * as spatialCatalogsActions from "../ducks/spatialCatalogs";
```

Comment:
I'm putting the comment here but it's about the Component in general, not that line.
Let's say I pick catalog A and entry A to filter on. Then, on top of the table you see the 2 filters being applied and you can click on them to delete one of them. If I delete the entry, well as discussed earlier the handler sends back an error saying the entry doesn't exist, and the table doesn't show up anymore. What could be much cleaner is that on delete of an element, both filtering are removed. 

However, I don't think that catching the deletes of the filters is something we do yet, so we would need to figure it out... It's not critical per say, but it does force you to reload SkyPortal, which can take a while sometimes, or at least change page and come back.

Reply:
https://github.com/skyportal/skyportal/issues/3780